### PR TITLE
merge csi-attack

### PR DESCRIPTION
**Pull request by jicahoo**
As title.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,9 @@
-#TODO
 FROM ubuntu:16.04
+LABEL description="Unity CSI Driver"
 
-COPY _output/csi-unity /csi-unity
+COPY ./_output/csi-unity /csi-unity
+
+RUN apt update
+RUN apt install -y nfs-common
```
Comment (Murray-LIANG) [inline Dockerfile]: Put two `RUN` in one line, like `RUN apt update && apt install -y nft-common` to save one layer of Docker image.
And consider to run apt clean to make the image smaller.

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 
 ENTRYPOINT ["/csi-unity"]
```
Comment (Murray-LIANG) [inline Dockerfile]: Append "/bin/bash" to `ENTRYPOINT` for easy debug of docker image.

**Review by Murray-LIANG — COMMENTED** [review]
Looks great.

File: Dockerfile (hunk 2 above)
Comment (jicahoo) [reply Dockerfile]: OK

File: Dockerfile (hunk 1 above)
Comment (jicahoo) [reply Dockerfile]: OK

**Comment by jicahoo** [comment]
Ryan approved this PR. :)

**Pull request merged**